db/dbSeed.js

```diff
@@ -1,7 +1,145 @@
+var faker = require('faker');
+const mysql = require('mysql');
+var connection = mysql.createConnection({
+  user: 'root',
+  password: '',
+  database: 'fec_Module_1'
+});
 
+connection.connect((err) => {
+  if (err) {
+    console.log('Make sure you load the db script in mysql before you try to connect', err);
+    return;
+  }
+  console.log('Successfully connected to the database!');
+});
```

Comment:
Is it necessary to connect the db in the seed?

Reply:
I guess I could just import the db/index.js file.

I just wanted to make sure that connection to the db was killed after seeding w/o impacting the actual server access to the db so I made it a separate instance.